services/distribution/src/main/resources/application.yaml

```diff
@@ -9,20 +9,33 @@ logging:
 
 services:
   distribution:
+    # The name of the distribution output file.
     output-file-name: index
+    # The number of days to retain diagnosis keys for both database persistency layer and files stored on the object store.
     retention-days: 14
+    # The number of minutes which diagnosis keys are allowed to be expired for (since submission) before they can be distributed.
```

Comment:
```suggestion
    # The number of minutes that diagnosis keys must have been expired for (since the end of the rolling interval window) before they can be distributed. Example: For a value of "expiry-policy-minutes: 120", the earliest point in time when a diagnosis key can be distributed is 26h after its "rolling-interval-start-number".
```

Reply:
I updated the first sentence to the revised one but IMO the more extended explanation should be reserved for the relevant .md documentation section.